config/settings.py

```diff
@@ -151,4 +151,15 @@
 
 LOGIN_REDIRECT_URL = 'catalog:index'
 LOGIN_URL = 'users:login'
-LOGOUT_REDIRECT_URL = 'catalog:index'
\ No newline at end of file
+LOGOUT_REDIRECT_URL = 'catalog:index'
+
+CACHES = {
+    "default":{
+        "BACKEND": 'django_redis.cache.RedisCache',
+        "LOCATION": 'redis://localhost:6379/1',
```

Comment:
Redis не всегда будет находиться на localhost:6379, поэтому правильнее адрес задавать через переменные окружения, например
```suggestion
        "LOCATION": 'redis://{REDIS_HOST}:{REDIS_PORT}/1',
```